app/views/admin/overview/index.html.erb

```diff
@@ -1,104 +1,211 @@
-            and
-            <a href="https://admin.forem.com/docs/advanced-customization/config/smtp-settings" data-action="click->ahoy#trackOverviewLink">email settings</a>
-          </li>
-          <li class="-ml-4">Join <a href="https://forem.dev" data-action="click->ahoy#trackOverviewLink">forem.dev</a> to learn from fellow Forem creators</li>
-        </ol>
-        <div class="align-center">
-          <a class="crayons-btn crayons-btn--ghost-brand" href="https://admin.forem.com" data-action="click->ahoy#trackOverviewLink">Visit Forem Admin Guide</a>
-        </div>
-      </div>
-    </section>
-    <h2 class="crayons-subtitle-1 mb-3">Analytics</h2>
-    <article class="grid s:grid-cols-2 gap-3 mb-4" data-controller="ahoy">
-      <section class="crayons-card">
-        <div class="crayons-card__body">
-          <h2 class="crayons-title"><%= @open_abuse_reports_count %></h2>
-          <p class="crayons-subtitle-4 color-base-70">
-            <a href=<%= admin_feedback_messages_path %>>
-              <%= "Open report".pluralize(@open_abuse_reports_count) %>
-            </a>
-          </p>
+</h1>
+
+<div class="flex flex-col gap-4">
+  <div class="flex flex-col m:flex-row gap-4">
+    <section class="crayons-card p-7 m:basis-2-3">
+      <header class="flex justify-between items-center mb-7">
+        <h2 class="crayons-subtitle-1">Analytics and trends</h2>
+        <div class="relative w-auto">
+          <button type="button" class="c-btn c-btn--icon-alone dropdown-trigger whitespace-nowrap" id="timeperiods-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
+            <%= crayons_icon_tag("calendar", title: "Timeframes", class: "c-btn__icon") %>
+          </button>
+          <form action="" id="timeperiods-dropdown" class="crayons-dropdown right-0 left-0 s:left-auto">
+            <fieldset class="mb-2">
+              <ul class="mb-0 p-2 flex flex-col gap-3">
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="7" id="7days" class="crayons-radio" checked />
+                  <label for="7days" class="crayons-field__label">Last 7 days</label>
+                </li>
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="14" id="14days" class="crayons-radio" />
+                  <label for="14days" class="crayons-field__label">Last 14 days</label>
+                </li>
+                <li class="crayons-field crayons-field--radio">
+                  <input type="radio" name="timeperiod" value="30" id="30days" class="crayons-radio" />
+                  <label for="30days" class="crayons-field__label">Last 30 days</label>
+                </li>
+              </ul>
+            </fieldset>
+            <button type="submit" class="c-btn c-btn--secondary w-100">Submit</button>
+          </form>
```

Comment:
I considered this being the most straightforward and simple way of switching timeframes.